tailwind.config.js

```diff
@@ -32,10 +62,27 @@ module.exports = {
           900: '#314156',
         },
       },
+
+      fontSize: {
+        xxSmall: 'var(----way-font-size-xx-small)',
```

Comment:
**issue**: Should be `----` instead of `--` (and for the following lines as well of course). In the deploy preview you can also see the font sizes don't display correctly.